pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -775,14 +775,6 @@ def parallel_devices(self) -> List[Union[torch.device, int]]:
     def num_processes(self) -> int:
         return self.devices if self.devices is not None else 1
 
-    @property
```

Comment:
just thinking if all connectors shall be protected classes...

Reply:
they already are: https://github.com/PyTorchLightning/pytorch-lightning/issues/9778